std/string.d

```diff
@@ -2939,6 +2939,30 @@ if (isForwardRange!Range && isSomeChar!(ElementEncodingType!Range) &&
     static import std.uni;
     import std.utf : decodeFront;
 
+    static if ((isDynamicArray!Range || (isRandomAccessRange!Range && hasLength!Range))
+        && !is(Unqual!(ElementEncodingType!Range) == dchar)
+        && __traits(compiles,
+            {
+                size_t s = input.length / 2;
+                input = input[s .. $];
+            }))
+    {{
```

Comment:
Enlighten me, why double curly braces here ?

Reply:
To create a scope. `static if` curly braces don't do that so I need a second pair.

Reply:
Sorry i must be tired but i don't see why a scope is needed here.

Reply:
Here it's not needed, I just prefer it.